vm/parser.ts

```diff
@@ -804,6 +804,11 @@ module J2ME {
     }
 
     get isNative(): boolean {
+      if (!release) {
+        if (Native[this.implKey]) {
+          return true;
+        }
+      }
```

Comment:
Nit: README.md should describe how to use this to override non-native methods with native implementations.